crates/algokit_transact_ffi/src/multisig.rs

```diff
@@ -0,0 +1,244 @@
+use crate::*;
+
+/// Representation of an Algorand multisignature signature.
+///
+/// This struct mirrors the Rust [`algokit_transact::MultisigSignature`] type, but uses FFI-safe types
+/// for cross-language interoperability.
```

Comment:
This will be exposed in the language binding, so probably don't want to mention FFI or Rust.

```suggestion
```